pkg/resources/preprocessor/workloads.go

```diff
@@ -20,10 +20,12 @@ import (
 )
 
 const (
-	configKey             = "config.yaml"
-	preprocessorVersion   = "v0.1.4-rc1-0.85.0"
-	preprocessorImageRepo = "ghcr.io/rancher-sandbox"
-	preprocessorImage     = "opni-otel-collector"
+	configKey = "config.yaml"
+
+	// TODO: Change back to opni-otel-collector image
+	preprocessorVersion   = "opensearch-exporter"
+	preprocessorImageRepo = "docker.io/jaehnri"
+	preprocessorImage     = "otel-collector"
```

Comment:
We should switch to a release of rancher-sandbox, unless something prevents it

Reply:
Yup, it is waiting for https://github.com/rancher-sandbox/opni-otel-collector/pull/12